pkg/operator/ceph/cluster/osd/labels.go

```diff
@@ -52,6 +52,11 @@ func (c *Cluster) getOSDLabels(osd OSDInfo, failureDomainValue string, portable
 	labels[OsdIdLabelKey] = stringID
 	labels[FailureDomainKey] = failureDomainValue
 	labels[portableKey] = strconv.FormatBool(portable)
+	if osd.DeviceClass == "hdd" {
```

Comment:
If we're basing this on the device class, I really think we should just directly add the `device-class` label. Then whoever looks at the label can interpret it themselves as rotational or not. 
```suggestion
	labels["device-class"] = osd.DeviceClass
```

Reply:
Done